### DISCUSSION TITLE
Report by minutes

### DISCUSSION THREAD

**Discussion by El-Tommy** [allinurl/goaccess#2234] [Q&A]
Today when I generate a report via Docker it is displayed for hours.
Is it possible to detail the number of hits per minute within a specific range?

**Comment by allinurl** [comment]
Are you referring to data within a specific panel, i.e., TIME DISTRIBUTION?

**Reply by El-Tommy** [reply]
I found the option --hour-spec=<hour|min> in the documentation, I think this will help me
